locales/en.default.schema.json

```diff
@@ -816,6 +816,19 @@
         "image_2": {
           "label": "Second image"
         },
+        "image_height": {
+          "label": "Image banner height",
+          "options__1": {
+            "label": "Small"
+          },
+          "options__2": {
+            "label": "Medium"
+          },
+          "options__3": {
+            "label": "Large"
+          },
+          "info": "For best results, use an image with a 2:3 aspect ratio. [Learn more](https://help.shopify.com/en/manual/shopify-admin/productivity-tools/image-editor#understanding-image-aspect-ratio)"
```

Comment:
Also I wonder, is 2:3 the aspect ratio we recommend ? Not 3:2 ? Since this section seem to be more for landscape images rather than portraits.

Reply:
Might want to check if we did this mistake on other sections where we mention 2:3 🤔  Taking a note to look for later!

Reply:
Aligned!